setup.py

```diff
@@ -73,6 +73,7 @@
         'ttfautohint-py',
         'ufolint',
         'unidecode',
+        'beziers>=0.1.0'
```

Comment:
if versions before 0.1.0 are not compatible, please add a comment here describing the reason. Otherwise, do not specify a minimum version requirement here on setup.py